### DISCUSSION TITLE
Study Short Title - CDE 11459812

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#69] [Metadata Field Ratification]
caDSR Link: [11459812](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11459812%20and%20ver_nr=1)

Definition: "The acronym or abbreviated form of the title for a research data collection. Example – GLIOMA01"

**Comment by GeoffLyle** [comment]
Accepted as of February 23, 2024.  

**Note:** We want to harmonize the _Study Short Title_ across Federation member data.  Will track the short titles being used on the Wiki page.

**Comment by GeoffLyle** [comment]
Merged concepts of **Study Acronym** and **Study Short Title** into _Study Short Title_.

This CDE is a rollup of concepts where a short title or an acronym was used.

**Comment by jpat1546** [comment]
Note: Study Short Name Text (11459812) has new version (same CDE): 
- prev (retired): https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11459812%20and%20ver_nr=1
- new (released): https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11459812%20and%20ver_nr=2

**Comment by GeoffLyle** [comment]
Per March 20, 2025 meeting:

Federation members agreed to use the updated version number (https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11459812%20and%20ver_nr=2).  This will require no changes to the harmonized data shared by Federation members.  This will require updates to the implementation code and documentation.  Will determine how these updates will be made in the API Workgroup.

**Comment by asafieva** [comment]
CDE 11459812 the latest version is 4.0 as of 06/05/2025. This is an updated link to the latest RELEASED CDE version:
[CDE 11459812  v4.0 Study Administration Study Short Name Text](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11459812%20and%20ver_nr=4).
